### PR TITLE
Updated appNotification TestApp

### PR DESCRIPTION
**Pull request by bernice-msft**
Changes included:
1) I made updates to the testApp, which allows for the displayAppNotification textbox to accept a string type URL.
2) I also included an e2e test data file, although we aren't conducting automated end to end testing, I set it up for future use/reference.
3) I also made a change to the message being sent to the webhubsdk via the message handler.

### CODE REVIEW

File: change/@microsoft-teams-js-cc1d35f9-c5f8-4d2b-9161-2d3c53d6d593.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added new `appNotification` capability for apps to send notifications to the user through the host",
```
Comment (lakhveerk) [inline change/@microsoft-teams-js-cc1d35f9-c5f8-4d2b-9161-2d3c53d6d593.json:3]: This change is not introducing app-Notification in teams-js. It is fixing the typos and bugs

File: apps/teams-test-app/e2e-test-data/appNotification.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "testCases":[
+   {
+    "title": "DisplayAppNotification API call- Success",
+    "type": "callResponse",
+    "boxSelector": "#box_checkAppNotificationCapability",
+    "inputValue": {
+      "title": "sampleAppNotificationTitle",
+      "content": "sampleAppNotficationContent",
+      "displayDurationInSeconds": 30,
+      "notificationActionUrl": "https://www.someAppNotificationUrl.com"
```
Comment (lakhveerk) [inline apps/teams-test-app/e2e-test-data/appNotification.json:18]: does this Url work when you try locally? I think it should not pass the domain validation

Comment (bernice-msft) [reply apps/teams-test-app/e2e-test-data/appNotification.json:18]: No it didn't, I've updated it to a URL which would pass the domain validation.

**Review by AE-MS — APPROVED** [review]
Approved but you should address @lakhveerk's comment about the changefile description, then I can approve it again.

**Review by AE-MS — APPROVED** [review]

**Pull request merged**